src/components/Articles.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../../api";
+import ArticleList from "./ArticleList";
+
+function Articles() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getArticles()
```

Comment:
it would be useful to have a loading pattern so the user gets feedback if the api call is taking a long time